docs/man/ctags.1.rst

```diff
@@ -214,25 +219,33 @@ Input/Output File Options
 
 ``--recurse[=yes|no]``
 	Recurse into directories encountered in the list of supplied files.
+
 	If the list of supplied files is empty and no file list is specified with
 	the ``-L`` option, then the current directory (i.e. '``.``') is assumed.
-	Symbolic links are followed by default. If you don't like these behaviors, either
+	Symbolic links are followed by default (See ``--links`` option). If you don't like these behaviors, either
 	explicitly specify the files or pipe the output of ``find(1)`` into
-	``ctags -L -`` instead. Note: This option is not supported on
-	all platforms at present. It is available if the output of the ``--help``
-	option includes this option. See, also, the ``--exclude`` and
+	``ctags -L -`` instead. See, also, the ``--exclude`` and
 	``--maxdepth`` to limit recursion.
 
+	Note: This option is not supported on
+	all platforms at present. It is available if the output of the ``--help``
+	option includes this option.
+
+.. TODO: Does not --list-features option support this?
+
```

Comment:
Thank you very much. This is an important suggestion. How do you think about introducing `.. TODO(code):` markup for the suggestions for improving the program code, not for the documents? So I can pick up this.